es-app/src/components/RatingComponent.h

```diff
@@ -23,7 +23,7 @@ class RatingComponent : public GuiComponent
 	void setValue(const std::string& value) override; // Should be a normalized float (in the range [0..1]) - if it's not, it will be clamped.
 
 	bool input(InputConfig* config, Input input) override;
-	void render(const Transform4x4f& parentTrans);
```

Comment:
Multiple compiler warnings:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/components/RatingComponent.
cpp:1:
/home/devel/temp/EmulationStation_Stable/es-app/src/components/RatingComponent.h:26:7: warning: 'rend
er' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]       
        void render(const Transform4x4f& parentTrans);
             ^
/home/devel/temp/EmulationStation_Stable/es-core/src/GuiComponent.h:41:15: note: overridden virtual f
unction is here
        virtual void render(const Transform4x4f& parentTrans);
                     ^
1 warning generated.